src/test/kfs_inference_service_test.cpp

```diff
@@ -0,0 +1,431 @@
+//*****************************************************************************
+// Copyright 2022 Intel Corporation
```

Comment:
Please rename file to kfs_metadata_test.cpp.

We don't want to put all inference service tests in one file.